src/core/tests/Test_MaintenanceWindow.py

```diff
@@ -79,5 +79,17 @@ def test_check_available_time_after_duration_complete(self):
         self.assertEqual(runtime.maintenance_window.is_package_install_time_available(), False)
         runtime.stop()
 
+    def test_get_percentage_maintenance_window_used(self):
```

Comment:
Add test for Exception ValueError if possible

Reply:
Added test